pages/storage_and_backup/object_storage/cold_archive_overview/guide.de-de.md

```diff
@@ -39,13 +45,15 @@ The service allows you to focus on building and deploying cloud applications whi
 
 The service is fully managed by OVHcloud and accessible through S3 API.
 
-**3 steps operation**
+**5 steps operation**
 
-1. Store your data in an Object Storage bucket first in RBX Region
-2. Archive it on tapes
-3. Restore your data or/and delete your archive
+1. Store your data in an Object Storage bucket first in RBX Region – this bucket is a temporary bucket,
+2. Archive it on tapes,
+3. Retrieve your data
+4. Decide to move or suppress the data from the temporary bucket
```

Comment:
for 4. and 5. could we try to mention there are post-archiving process (move/delete actions).